src/Serilog/Serilog.csproj

```diff
@@ -28,19 +30,19 @@
   </PropertyGroup>
 
   <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.1' ">
-    <DefineConstants>$(DefineConstants);FEATURE_DEFAULT_INTERFACE;FEATURE_SPAN;FEATURE_ASYNCDISPOSABLE;FEATURE_DICTIONARYTRYADD</DefineConstants>
+    <DefineConstants>$(DefineConstants);FEATURE_DEFAULT_INTERFACE;FEATURE_SPAN;FEATURE_ITUPLE;FEATURE_ASYNCDISPOSABLE;FEATURE_DICTIONARYTRYADD</DefineConstants>
```

Comment:
`FEATURE_ITUPLE` is not needed anymore

Reply:
What should I use instead?

Reply:
Nothing, this is merge artifact.

Reply:
I don't think that's correct; there's still a reference to it in `PropertyValueConverter`, and the CSPROJ targets .NET 4.6 where it's unsupported.

https://github.com/serilog/serilog/blob/dev/src/Serilog/Capturing/PropertyValueConverter.cs#L249

Still some work to do once we've finalized framework version support.

Reply:
So I confused it with another feature.